packages/backend/src/dbt/profiles.ts

```diff
@@ -93,6 +93,24 @@ const credentialsTarget = (
                     [envVar('password')]: credentials.password,
                 },
             };
+        case WarehouseTypes.TRINO:
+            return {
+                target: {
+                    type: credentials.type,
+                    host: credentials.host,
+                    method: 'ldap',
+                    user: credentials.user,
+                    password: credentials.password,
```

Comment:
These two should be used via env vars that are defined in the `environment` in line 109. This is because the target will be used to generate a dbt_profile file and we don't want secrets in there.
```suggestion
                    user: envVarReference('user'),
                    password: envVarReference('password'),
```